okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/RetryInterceptor.java

```diff
@@ -60,7 +61,7 @@ public Response intercept(Chain chain) throws IOException {
                 }
             }
             if (!successful) {
-                log.error("Request to url {} failed on attempt {}", request.url(), numTries);
+                log.error("Request to url {} failed on attempt {}", request.url(), SafeArg.of("numTries", numTries));
```

Comment:
log message should get cleaned up to:

    "Failed request to url"

Reply:
why? don't like the unsafe url being logged?